.github/workflows/build.yml

```diff
@@ -35,9 +43,9 @@ jobs:
         with:
           shared-key: ${{ runner.os }}-cargo
 
-      - uses: taiki-e/install-action@v1
+      - uses: taiki-e/install-action@v2
         with:
-          tool: cargo-hack
+          tool: cargo-hack,cargo-llvm-cov,nextest
```

Comment:
have we verified if `nextest` is really beneficial? I can see the CI runtime is almost same, can you just confirm once if `nextest` is adding anything to value?

Reply:
nextest isn't running [doctest](https://github.com/nextest-rs/nextest/issues/16), which means we are just ignoring them.

what features do we want exactly though? just curious. are we seeing improvement in test suite runtime? Other features require some configuration as well right?

Reply:
> which means we are just ignoring them

Neither llvm-cov supports coverage over doctests. We can test the docs separately for now, and when nextest starts supporting them, we'll include them too.

Reply:
> not including doc tests still bugs me a lot and make me doubt the accuracy of the report haha ( especially in our case )

Will add it as a separate step.